docs/theme/build.js

```diff
@@ -0,0 +1,215 @@
+import StyleDictionary from 'style-dictionary';
+import { parse, formatHex, formatHex8, oklch, formatCss } from 'culori';
```

Comment:
So, **this is not a blocker**, but it might be nice to use colorjs.io everywhere, I've been recently going over a lot of projects related to design tokens (incl. style-dictionary) and noticed there is a lot of usages of culori, chromajs, tinycolor2 and so far I haven't encountered anything colorjs.io doesn't handle, so it's been kinda worthwhile migrating everything to it and not use many different color libs at the same time.